spec/stat_tracker_spec.rb

```diff
@@ -38,4 +38,24 @@
   it "can calculate the pecentage of games that end up in a tie" do
     expect(@stat_tracker_1.percentage_ties).to eq(0.08)
   end
+
+  it "can count games in each season" do
+    expect(@stat_tracker_1.count_of_games_by_season).to eq({"20122013"=>11, "20132014"=>2})
+  end
+
+  #average_goals_per_game
```

Comment:
I love how we went through this process as a sync session. We were able to see the progression of the tests in real time and contribute together.